app/src/main/java/com/example/kuba10/mypokemonplaces/RESTutils/RetrofitConnection.java

```diff
@@ -1,91 +0,0 @@
-package com.example.kuba10.mypokemonplaces.RESTutils;
-
-import android.util.Log;
-
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.jakewharton.retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
-
-import java.util.ArrayList;
-import java.util.List;
-
-import io.reactivex.Observer;
-import io.reactivex.android.schedulers.AndroidSchedulers;
-import io.reactivex.disposables.Disposable;
-import io.reactivex.schedulers.Schedulers;
-import retrofit2.Retrofit;
-import retrofit2.converter.gson.GsonConverterFactory;
-
-
-public class RetrofitConnection {
-
-    private Retrofit retrofit;
-    private PokeApiService pokeRetrofit;
-    ArrayList<PokemonGo> pokemonList;
```

Comment:
- To jest klasa `RetrofitConnection` - powinny być tu tylko rzeczy związane z retrofitem. Nazwij ją jakoś inaczej
- Retrofit uwtórz w konstruktorze tej klasy (wtedy zrób `private final Retrofit retrofit`)
- To samo z serwisem
- To samo z listą